app/server/src/routes/badge/badge.validate.js

```diff
@@ -0,0 +1,8 @@
+const { body, query } = require("express-validator");
+
+exports.validate = (method) => {
+    switch (method) {
+
+        case "getCall":
```

Comment:
We might want validation for post requests, at least for existence. In this case, createBadge